src/components/dialogs/FileCorrectionDialog.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="file-correction-dialog">
+    <v-card>
+      <v-card-title class="pt-8">Correction Filing</v-card-title>
+
+      <v-card-text class="font-15">
+        <p>Select who caused the correction. If client, completing party information and certification
+          will be required. If staff, completing party information and certification will not be required.</p>
+        <v-radio-group v-model="correctionOption">
+          <v-radio id="correct-client-radio" class="mb-0 pt-2" label="Client Error" :value="correctionOptions.CLIENT" />
+          <v-radio id="correct-staff-radio" class="mb-0 pt-2" label="Staff Error" :value="correctionOptions.STAFF" />
+        </v-radio-group>
+        <template v-if="!hasChosenCorrection">
+          <p class="font-15 option-error">Choose one option to proceed</p>
+        </template>
+      </v-card-text>
+
+      <v-card-actions class="py-10">
+        <v-btn outlined large
+          class="mr-2"
+          color="primary"
+          id="dialog-exit-button"
+          @click="exit()"
+        > Cancel </v-btn>
+        <v-btn depressed large
+          color="primary"
+          id="dialog-start-button"
+          @click="checkToStart()"
+        >Start Correction</v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit, Watch } from 'vue-property-decorator'
+import { mapGetters } from 'vuex'
+import { ContactInfo } from '@/components/common'
+
+@Component({
+  computed: {
+    // Property definition for runtime environment.
+    ...mapGetters(['isRoleStaff'])
+  },
+  components: { ContactInfo }
+})
+export default class FileCorrectionDialog extends Vue {
+  // Initialize hasChosenCorrection.
+  private hasChosenCorrection = true
+
+  /** Radio group model property. */
+  private correctionOptions = {
+    NONE: 'NONE',
+    CLIENT: 'CLIENT',
+    STAFF: 'STAFF'
+  }
```

Comment:
This should be an enum and should be used in other places in this project.

Also, I see the value here is emitted as a string, is received as a string, and that string is appended to a route name or something. Instead, to make good use of typings and to help prevent issues, please keep this as a enum all the way until the route name is created. This is also the concept of keeping things where they belong; specifically, THIS code should not care what the route name will be.